### PR TITLE
Add the property isLoaded to composers.

### PR DESCRIPTION
**Pull request by thibaultzanini**

### CODE REVIEW

File: ui/overlay.reel/overlay.js [hunk 1]
```diff
@@ -171,6 +154,8 @@ exports.Overlay = Component.specialize( /** @lends Overlay.prototype # */ {
 
                 _window = this.element.ownerDocument.defaultView;
                 _window.addEventListener("resize", this);
+
+                this._pressComposer = new PressComposer();
```
Comment (marchant) [inline ui/overlay.reel/overlay.js]: This should be in prepareForActivationEvent, not in enterDocument

Comment (thibaultzanini) [inline ui/overlay.reel/overlay.js]: @benoitmarchant I don't think so, given that it's an overlay we need to be able to know if an user click outside an overlay in order to hide that overlay.

Comment (thibaultzanini) [inline ui/overlay.reel/overlay.js]: We could instead create it when this overlay is shown for the first time.

File: ui/component.js [hunk 2]
```diff
@@ -2385,15 +2385,44 @@ var Component = exports.Component = Target.specialize( /** @lends Component.prot
...
     },
 
     /**
+     * Load a Composer
+     * @function
+     * @param {Composer} composer
+     */
+    loadComposer: {
+        value: function (composer) {
+            if (this.composerList.indexOf(composer) > -1) { // [Need discussion] -> composer.component === this
```
Comment (thibaultzanini) [inline ui/component.js]: FYI @marchant need discussion.

**Pull request merged**